test/integration/sample.profiles.yml

```diff
@@ -0,0 +1,42 @@
+# HEY! This file is used in the tsql_utils integrations tests with CircleCI.
+# You should __NEVER__ check credentials into version control. Thanks for reading :)
+
+config:
+  send_anonymous_usage_stats: False
+  use_colors: True
+
+defaults:
+  sqlserver: &sqlserver
+    type: sqlserver
+    driver: "ODBC Driver 17 for SQL Server"
+    schema: "dbt_cnxn_test"
+    port: 1433
+    threads: 8
+
+integration_tests:
+  target: sqlserver_local_userpass
+  outputs:
+    sqlserver_local_userpass: &sqlserver-local
+      <<: *sqlserver
+      host: localhost
+      database: msdb
+      username: SA
+      password: 5atyaNadella
+    sqlserver_local_encrypt:
+      <<: *sqlserver-local
+      encrypt: yes
+      trust_cert: yes
+    azuresql_sqlcred: &azuresql
+      <<: *sqlserver
```

Comment:
For my understanding, is this a way to extend de above mentioned profiles, the ones with `&<some name>`?

Reply:
that's right!

Reply:
they're called YAML anchors, and I'm on the fence about them legibility-wise. Seems pretty easy to go overboard on. how would you rate the readability of this YAML file, @JCZuurmond? 
https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/